ui/ui-components/components/PageHeader.tsx

```diff
@@ -32,7 +34,7 @@ export default function PageHeader({
                   email={email}
                 />
               ) : iconType === "grouparooModel" ? (
-                <ModelIcon model={{ icon }} size="7x" />
```

Comment:
I felt 7x was just to big so I downgraded to a smaller size that fits closer to the heading. This change does impact the rest of the pages.